recognition/include/pcl/recognition/dense_quantized_multi_mod_template.h

```diff
@@ -39,6 +39,7 @@
 
 #include <vector>
 
+#include <pcl/common/common.h>
```

Comment:
Is this the most granular header that you could be using?

Reply:
Oh! I meant to include exceptions.h, I'll change it